content/influxdb/v2.0/reference/cli/influx/v1/auth/create.md

```diff
@@ -0,0 +1,38 @@
+---
+title: influx v1 auth create
+description: >
+  The `influx v1 auth create` command creates an authorization in InfluxDB.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 auth create
+    parent: influx v1 auth
+weight: 101
+influxdb/v2.0/tags: [authorization]
+---
+
+The `influx v1 auth create` creates an authorization in InfluxDB.
```

Comment:
When would they use create versus auth?

Reply:
My understanding is that `influx v1 auth` on its own is currently just a placeholder for the subcommands.